tests/integration.txtar

```diff
@@ -6,6 +6,12 @@
 # move on the testcases directory
 cd testcases/example-app/
 
+# test it doesn't panic when the file is not executable
```

Comment:
can you move it right above the `# setting up test application`?

Reply:
Done in https://github.com/alegrey91/harpoon/pull/60/commits/3522f20b126c86e0e05ce91bb176d23145c97771